3_Print_formatting.ipynb

```diff
@@ -177,7 +177,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3.4 Special characters  \n",
+    "## 3.4 What's the value of my variable?\n",
+    "Sometimes you will want to print the name of a variable alongside its value, for example during debugging.\n",
+    "Intuitively you might do this like so:\n",
+    "```python\n",
+    "myvar = pi * 2.4**2\n",
+    "print(f\"mvar={myvar}\")\n",
```

Comment:
```suggestion
    "print(f\"myvar={myvar}\")\n",
```

Reply:
This nicely illustrates the point I was making above!